sierra-previewer/src/main/resources/org/httprpc/sierra/previewer/MainFrame.xml

```diff
@@ -12,26 +12,18 @@
         </menu>
         <menu text="About">
             <menu-item name="aboutItem" text="About Previewer"/>
-        </menu>        
+        </menu>
     </menu-bar>
 
-		<split-pane name="splitPane" weight="1">  
-           <scroll-pane name="editorScrollPane" weight="1">
-              <label text="XML will appear here."
-                   horizontalAlignment="center"
-                   verticalAlignment="center"
-                   foreground="gray"
-                   weight="1"/>
-             </scroll-pane>                   
-   		  <column-panel name="previewPanel" padding="5, 5, 5, 5" weight="1">
-            <label text="Preview will appear here."
-                   horizontalAlignment="center"
-                   verticalAlignment="center"
-                   foreground="gray"
-                   weight="1"/>
-            </column-panel>
-		 </split-pane>
-               
-    <label name="filePathLabel" text="" padding="4, 6, 4, 6"/>
+    <split-pane name="splitPane" resizeWeight="0.5" weight="1">
+        <scroll-pane weight="1">
+            <syntax-text-area name="editorPane" columns="80" rows="25"
```

Comment:
Declare editor in markup.

Reply:
I just realized that this will make it seem like "syntax-text-area" is part of the standard distribution, which it is not. So maybe this is not a good idea after all.

Reply:
Yeah agree - only valid widgets should be in the xml - even if they are placeholders to be replaced at runtime